BRANDING.MD

```diff
@@ -0,0 +1,24 @@
+## Knative Branding Guidelines
+
+
+The Knative logo is made available here with the following guidelines:
+
+ **You may:**
+ - use the logo for non-commercial uses such as t-shirts and stickers.
+ - use the logo to link to knative.dev
+ - use the logo in a blog post or news article about Knative
+ - use the logo to advertise that your product has support for Knative [Note that at this time such usage does *not* indicate conformance with any standards or requirements]
+
+**You may not:**
+
+ - Use the Knative logo as your application's icon or logo
+ - Create a modified version of the Knative logo (except for non-commercial uses such as t-shirts and stickers)
+ - Integrate the Knative logo into your logo
+ - Change the Knative logo's colors
```

Comment:
Isn't this covered by `Create a modified version of the Knative logo`?

Reply:
The intent is that we allow it for stickers, etc. but not for other purposes.

Reply:
Yes, but it explicitly states that it's prohibited to
`Create a modified version of the Knative logo (except for non-commercial uses such as t-shirts and stickers)`
so `Change the Knative logo's colors` is subset of `modified version`?